tests/test_server.py

```diff
@@ -0,0 +1,105 @@
+import asyncio
+import unittest
+import json
+
+from unittest import mock
+
+
+class TestServer(unittest.TestCase):
+    MockedRequestHandler = None
+    srv = None
+    m = None
+    m_eof = None
```

Comment:
It's my mistake. Should I use setUpClass() or __init__() or setUp() to initialize self variables in the test class?